.github/workflows/auto-merge-bot.yml

```diff
@@ -0,0 +1,19 @@
+on: [pull_request]
+
+jobs:
+  auto_merge_bot:
+    runs-on: ubuntu-latest
+    name: EIP Auto-Merge Bot
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js Environment
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: auto-merge-bot
+        uses: alita-moore/EIP-Bot@master # Uses an action in the root directory
```

Comment:
Can we have this point at a particular commit sha?  From a security standpoint, that would let me sleep better at night knowing that the code executing against this repository can't change without a PR to this repository.

Reply:
Yeah that's fine. I'll point it toward a versioning system!